design-tokens/tokens/primitive/primitives.default.json

```diff
@@ -349,6 +409,18 @@
           }
         }
       },
+      "25600": {
+        "$type": "number",
+        "$value": 1024,
+        "$description": "",
+        "$extensions": {
+          "com.figma": {
+            "hiddenFromPublishing": false,
+            "scopes": [],
+            "codeSyntax": {}
+          }
+        }
+      },
       "38400": {
```

Comment:
The primitive token to value pattern seems to break here. If I understand how these primitive tokens are being built up, there is a ratio of `25` between the token and value (e.g. 600 / 25 = 24, 25600 / 25 = 1024).

However, 38400 & 76800 seem to break this pattern. 38400 / 25 = 1536, not 1152 as currently defined; 76800 / 25 = 3072, not 1536.

Do we need to:
- [ ] add 28800 --> 1152
- [ ] modify 38400 --> 1536
- [ ] add 51200 --> 2048
- [ ] modify 78600 --> 3072

Reply:
Good catch. Yes 38400 --> 1536. I don't believe we have use for 1152, 2048, or 3072. 

I've adjusted 38400, but not added the others.

Reply:
I'm good with omitting those for now. 

We don't need the 1152 because we are just hardcoding that in the HPE theme, correct?

Reply:
Exactly. 👍